argus/scenario.py

```diff
@@ -418,6 +418,61 @@ def get_remote_client(self, username=None, password=None,
     remote_client = util.cached_property(get_remote_client, 'remote_client')
 
 
+class RescueWindowsScenario(BaseWindowsScenario):
+    """Instance rescue Windows-based scenario."""
+
+    def rescue_server(self):
+        admin_pass = self._image.default_ci_password
+        self._servers_client.rescue_server(self._server['id'],
+                                           adminPass=admin_pass)
+        self._servers_client.wait_for_server_status(self._server['id'],
+                                                    'RESCUE')
+
+    def unrescue_server(self):
+        self._servers_client.unrescue_server(self._server['id'])
+        self._servers_client.wait_for_server_status(self._server['id'],
+                                                    'ACTIVE')
+
+    def run_test_suite(self):
+        testloader = unittest.TestLoader()
+        suite = unittest.TestSuite()
+        for test_class in self._test_classes:
+            testnames = testloader.getTestCaseNames(test_class)
+            for name in testnames:
+                suite.addTest(test_class(name,
+                                         manager=self,
+                                         service_type=self._service_type,
+                                         image=self._image))
+        self._result = unittest.TestResult()
+        return suite.run(self._result)
+
+    def run(self):
+        """Run the tests from the underlying test class.
+
+        This will start a new instance and prepare it using the recipe.
+        It will return a list of test results.
+        """
+
+        self._prepare_run()
+
+        try:
+            # prepare the instance
+            self._setup()
+            # save the output
+            self._save_instance_output()
+
+            # run the tests
+            LOG.info("Running tests...")
+
+            return self.run_test_suite()
+        except:
+            import traceback
+            LOG.debug("".join(traceback.format_exception(*sys.exc_info())))
+            raise
```

Comment:
This needs to be removed, since the error will be shown as a failure when the test suite fails, so it doesn't add any real value.